tests/unit_tests/hub/test_hub_install.py

```diff
@@ -16,18 +16,19 @@
 )
 class TestInstall:
     def setup_method(self):
-        self.manifest = ModuleManifest.from_dict(
+        self.manifest = Manifest.from_dict(
             {
                 "id": "id",
                 "name": "name",
                 "author": {"name": "me", "email": "me@me.me"},
                 "maintainers": [],
                 "repository": {"url": "some-repo"},
                 "namespace": "guardrails",
-                "package_name": "test-validator",
-                "module_name": "test_validator",
+                "packageName": "test-validator",
```

Comment:
@CalebCourier thought about this at lunch had to update these to get them passing but wont the hub return manifests with _ and not camel. cc @AlejandroEsquivel for 2 cents unless im mistaken

Reply:
@dtam the manifest properties should all be camelCase now: https://github.com/guardrails-ai/guardrails-hub/blob/playground-support/validators/guardrails/detect_pii/manifest.json#L19-L20

Reply:
In fact, they always have been on main https://github.com/guardrails-ai/guardrails-hub/blob/main/validators/guardrails/detect_pii/manifest.json#L45-L46

Reply:
Seems to be camel case https://hub.api.guardrailsai.com/validator-manifests/scb-10x/correct_language

Reply:
just kidding im guessing this is what the hub updates on this pr does. good times